apps/judicial-system/web/src/routes/Defender/CaseOverview.tsx

```diff
@@ -71,233 +77,254 @@ export const CaseOverview: React.FC = () => {
-                        signingDate={workingCase.courtRecordSignatureDate}
-                      />
-                    ) : null}
-                  </PdfButton>
-                  <PdfButton
-                    renderAs="row"
-                    caseId={workingCase.id}
-                    title={formatMessage(core.pdfButtonRuling)}
-                    pdfType={'ruling/limitedAccess'}
-                  >
-                    <SignedDocument
-                      signatory={workingCase.judge?.name}
-                      signingDate={workingCase.rulingDate}
-                    />
-                  </PdfButton>
-                </>
-              )}
-            </Stack>
+          {completedCaseStates.includes(workingCase.state) && (
+            <Box marginBottom={6}>
+              <BlueBox>
+                <Box marginBottom={2} textAlign="center">
+                  <Text as="h3" variant="h3">
+                    {formatMessage(m.conclusionHeading)}
+                  </Text>
+                </Box>
+                <Box marginBottom={3}>
+                  <Box marginTop={1}>
+                    <Text variant="intro">{workingCase.conclusion}</Text>
+                  </Box>
+                </Box>
+                <Box marginBottom={1} textAlign="center">
+                  <Text variant="h4">{workingCase?.judge?.name}</Text>
+                </Box>
+              </BlueBox>
+            </Box>
+          )}
+          <Box marginBottom={10}>
+            <Text as="h3" variant="h3" marginBottom={3}>
+              {formatMessage(m.documentHeading)}
+            </Text>
+            <Box marginBottom={2}>
+              <Stack space={2} dividers>
+                <PdfButton
+                  renderAs="row"
+                  caseId={workingCase.id}
+                  title={formatMessage(core.pdfButtonRequest)}
+                  pdfType={'request/limitedAccess'}
+                />
+                {completedCaseStates.includes(workingCase.state) && (
+                  <>
+                    <PdfButton
+                      renderAs="row"
+                      caseId={workingCase.id}
+                      title={formatMessage(core.pdfButtonRulingShortVersion)}
+                      pdfType={'courtRecord/limitedAccess'}
+                    >
+                      {workingCase.courtRecordSignatory ? (
+                        <SignedDocument
+                          signatory={workingCase.courtRecordSignatory.name}
+                          signingDate={workingCase.courtRecordSignatureDate}
+                        />
+                      ) : null}
+                    </PdfButton>
+                    <PdfButton
+                      renderAs="row"
+                      caseId={workingCase.id}
+                      title={formatMessage(core.pdfButtonRuling)}
+                      pdfType={'ruling/limitedAccess'}
+                    >
+                      {workingCase.rulingDate ? (
```

Comment:
This diff is awful to read, it makes it seem like I changed more than I actually dod, so just for clarity this is what I added in terms of handling the signed document logic.